rental_options.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'person_options'
+require_relative 'book_options'
+require_relative 'rental'
+
+class RentalOptions
+  def initialize(books, people)
+    @rentals = []
+    @books = books
+    @people = people
+  end
+
+  def create_a_rental
+    puts "\nPlease, select a book from the following list by number: "
+    list_all_books
+    selected_book = gets.chomp.to_i
+
+    puts "\nSelect a person from the following list by number (not id)"
+    list_all_people
+    selected_person = gets.chomp.to_i
+
+    print 'Date: '
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you provide to users a format of the date to be added while completing the above option. This will help to improve the User experience and to add correct data. For example this to be:
```ruby
...
 print 'Date(YYYY-MM-DD): '
```

Reply:
Hello @tresorsawasawa, I totally agree. I'm doing the respective changes, thanks for the review !!